pettingzoo/utils/wrappers/terminate_illegal.py

```diff
@@ -39,7 +42,7 @@ def step(self, action):
             self.terminations[self.agent_selection]
             or self.truncations[self.agent_selection]
         ):
-            self._was_dead_step(action)
+            self._was_dead_step(None)
```

Comment:
This is meant to be a type hint fixing PR, why was this change made? Imo revert it unless there's a good reason otherwise

Reply:
Reverted